website/.unbroken_exclusions

```diff
@@ -1,5 +1,6 @@
 !node_modules
 !blog
+!versioned_docs/version-0.60/e2e-test-more-about.md
```

Comment:
Is this correct? Does there need to be a new one for the *-coreapp.md file?

Reply:
no, blogs are not checked (see line number 2); 
this is needed because in the 0.60 e2e testing doc, we have a link to a wdio package and that package has been removed in latest versions of wdio, so the link is now broken. This gets our CI passing again.